iam/hooks.py

```diff
@@ -257,9 +257,12 @@ def was_verified_user(cog, obj, *func_args, **func_kwargs):
         obj = obj.author
     member = get_member(cog.bot, obj)
     if member is not None and VER_ROLE in get_role_ids(member):
-        return
-    member_data = cog.db.get_member_data(obj.id)
-    if member_data is None or not member_data[MemberKey.ID_VER]:
+        return True, None
+    try:
+        member_data = cog.db.get_member_data(obj.id)
+    except MemberNotFound:
+        return False, "You must be verified to do that."
+    if not member_data[MemberKey.ID_VER]:
         return False, "You must be verified to do that."
```

Comment:
I never like functions returning more than one thing like this. Generally if you have to do this, you should be creating a custom class that has both of these values as attributes.

Reply:
In other words, Go != Python